db/migrations/02_add_date_to_peeps.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE peeps ADD COLUMN date DATE;
```

Comment:
Also the use of the DATE syntax to store the current date is something I didn't implement either and something I can take away from this, good job!